### ISSUE TITLE
Add ACM DL link for publications with no DOI

### ISSUE THREAD

**Issue by khatchad** [backtracking/bibtex2html#16]
If the DOI is not present but an `acmid` is, then please add a link to the ACM DL page. Currently, no link is generated.

**Comment by claudemarche** [comment]
the option 
   -f acmid
or
   -nf acmid "ACM DL"
should do what you want

**Comment by khatchad** [comment]
Not quite. I get a link, but it's to a local file that doesn't exist. The link looks to be to a file in the working directory with the name of the ACM ID, e.g., file:///home/rk1424/Documents/Curriculum%20Vitae/2892681.

On Tue, 2017-11-28 at 08:38 +0000, Claude Marché wrote:

the option
-f acmid
or
-nf acmid "ACM DL"
should do what you want
